frontend/src/components/app/app.tsx

```diff
@@ -12,6 +13,7 @@ const App: FC = () => {
         <Route path={AppRoute.ROOT} element="Root" />
         <Route path={AppRoute.SIGN_UP} element={<Auth />} />
         <Route path={AppRoute.SIGN_IN} element={<Auth />} />
+        <Route path={AppRoute.UAM} element={<Uam />} />
```

Comment:
```suggestion
        <Route path={AppRoute.UAM} element={<AuthorizedWrapper><Uam /></AuthorizedWrapper>} />
```

Reply:
Seems that Oleksandr has not done it yet (Protected route component), if you mean that

Reply:
Oh, Oksana did that task, I will improve this fix